index.js

```diff
@@ -13,6 +13,7 @@ import {
 import cloneDeep from 'lodash/cloneDeep';
 import clone from 'lodash/clone';
 import isEqual from 'lodash/isEqual';
+import Annotation from './MapboxAnnotation';
```

Comment:
Can we name the class `MapboxAnnotation` or the file `Annotation` to remove this mismatch?